packages/app-project/src/components/Announcements/Announcements.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react'
 
-import ResearcherAnnouncement from './components/ProjectAnnouncement'
+import ProjectAnnouncement from './components/ProjectAnnouncement'
+import FinishedAnnouncement from './components/FinishedAnnouncement'
 
 function Announcements () {
   return (
     <>
-      <ResearcherAnnouncement />
+      <ProjectAnnouncement />
```

Comment:
Is `<FinishedAnnouncement />` supposed to be in here, next to `<ProjectAnnouncement />`?

Reply:
Ah, ignore my comment. 

> This is not added to be viewable in context of a project yet because the SEE RESULTS link isn't not finished yet.